### PR TITLE
Added Monthly Inputs in HTML

### PR DESCRIPTION
**Pull request by smith-taher**
@janelleany 
@paulwooan 
Please review the Monthly Input fields.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
     </div>
     
     <h3>Hello, yourName</h3>
+
+     <!--monthly income input  -->
+     <div class="goalsAndExpenses">
```
Comment (janelleany) [inline index.html]: Is there a .goalsAndExpenses class in the css? If not I would delete this class for now.

File: index.html [hunk 2]
```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
...
+        <label for="MonthlyIncome">Monthly Income</label>
+        <input class="form-control " name="income" id="monthlyIncome" value="" autofocus placeholder="Monthly Income">
```
Comment (janelleany) [inline index.html]: please add 'required' attribute in this input's opening tag

File: index.html [hunk 3]
```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
...
+    </div>
+
+    <!--monthly savings goal input  -->
+    <div class="monthlySavings">
```
Comment (janelleany) [inline index.html]: if no .monthlySavings class in the css please delete for now

File: index.html [hunk 4]
```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
...
+        <label for="savingsGoal">Savings Goal</label>
+        <input class="form-control" type="savings" name="savingsGoal" id="savingsGoal" value="" placeholder="Monthly Savings Goal">
```
Comment (janelleany) [inline index.html]: please add 'required' attribute to input's opening tag

**Review by janelleany — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (smith-taher) [reply index.html]: The class has been created in HTML to be referenced in CSS later on.

File: index.html (hunk 3 above)
Comment (smith-taher) [reply index.html]: The class has been created in HTML to be referenced in CSS later on.

File: index.html (hunk 2 above)
Comment (smith-taher) [reply index.html]: Working on it now. Great idea!

File: index.html (hunk 4 above)
Comment (smith-taher) [reply index.html]: Working on it now.

**Review by janelleany — APPROVED** [review]

**Comment by smith-taher** [comment]
Joel's first merge!

**Pull request merged**